docs/kafka/rhoas-cli-getting-started-kafka/README.adoc

```diff
@@ -128,42 +129,50 @@ A Kafka instance includes a Kafka cluster, bootstrap server, and other required
 
 .Prerequisites
 
-* You are logged in to `rhoas`.
+* You're logged in to `rhoas`.
 
 .Procedure
 
 . Enter the following command to create a Kafka instance with default values.
 +
 --
-This example creates a Kafka instance called `my-kafka`.
-
 .Creating a Kafka instance
 [source,shell]
 ----
-$ rhoas kafka create --name my-kafka
+$ rhoas kafka create
 ----
+--
+
+. Enter Name: `my-kafka`.
+. Use the arrows on your keyboard to select your OpenShift cluster for your Kafka instance.
+. Select an instance size of 1 (x1) or 2 (x2) streaming units. 
++
+--
+A streaming unit determines the default maximum capacity of a Kafka instance. An instance with a larger size can handle higher loads and process more events, has more storage, and can handle more clients and connections.
+
+The CLI displays information about the created instance including billing model, cloud provider, cluster id of the cluster the instance is provisioned on, and instance type.
 
 CLI uses service contexts to group service instances against which commands are executed.
 The created Kafka instance is set automatically in the current context.
 
 [NOTE]
 ====
-If you do not want to use the default values,
-enter the following command: `rhoas kafka create`.
-You will be prompted to enter the `Name`, `Cloud Provider`, and `Cloud Region` for the Kafka instance.
+Alternatively, to provision a Kafka instance by specifying the kafka name and cluster-id, enter the following command: `rhoas kafka create --name my-kafka --cluster-id <CLUSTER_ID>`. This takes on the default value size 1 streaming unit.
```

Comment:
```suggestion
Alternatively, to provision a Kafka instance by specifying the Kafka name and cluster ID, enter the following command: `rhoas kafka create --name my-kafka --cluster-id <CLUSTER_ID>`. Replace <CLUSTER_ID> with your cluster ID. Using this command creates an instance size with the default value size of 1 streaming unit.
```

Reply:
@fmcdonal. As per [user-replaced values guidelines](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values),  the value needs to be italicized. So the command should be: `rhoas kafka create --name my-kafka --cluster-id _<CLUSTER_ID>_`.

Reply:
Hi @fmcdonal. This value needs to be italicized.